web/components/avatar.tsx

```diff
@@ -21,32 +20,28 @@ export function Avatar(props: {
           e.stopPropagation()
           Router.push(`/${username}`)
         }
-  return (
-    <div
+
+  // there can be no avatar URL or username in the feed, we show a "submit comment"
+  // item with a fake grey user circle guy even if you aren't signed in
```

Comment:
I think it's relevant because my common sense says, "Now that everything always has an avatar, why would we have a code path for displaying a grey non-avatar? Is that obsolete?" So I like to see the answer "no, it's not obsolete, we use it in this one weird place that is easy to forget."

If you feel strongly I will get rid of it but it's an answer to a real question that was in my mind when I first read this code.